core/src/main/resources/hudson/PluginManager/advanced.jelly

```diff
@@ -37,22 +37,11 @@ THE SOFTWARE.
         <l:main-panel>
           <l:app-bar title="${%Advanced settings}"/>
 
+          <div class="alert alert-info">
+              <strong>${%The Proxy configuration form has been moved to} <a href="${rootURL}/manage/configure">Configure System page</a></strong>
```

Comment:
this breaks localisation as 'Configure System page' can't be localised. I think you want something like this and moving the link into property bundle:
```suggestion
              <strong>${%proxyMovedBlurb("${rootURL}/manage/configure")}</strong>
```

Reply:
this is not the configure system page but the plugin manager?
if this does not work then is the [other](https://github.com/jenkinsci/jenkins/blob/310bd6ac11939f525d055193b6d5d91ddc6ffcef/core/src/main/resources/hudson/PluginManager/advanced.jelly#L51) [localisation](https://github.com/jenkinsci/jenkins/blob/310bd6ac11939f525d055193b6d5d91ddc6ffcef/core/src/main/resources/hudson/PluginManager/advanced.jelly#L56) [in](https://github.com/jenkinsci/jenkins/blob/310bd6ac11939f525d055193b6d5d91ddc6ffcef/core/src/main/resources/hudson/PluginManager/advanced.jelly#L61) this page also broken?

Reply:
fixed with 48d3fcf691